src/code/components/clutch-view.js

```diff
@@ -0,0 +1,84 @@
+import React, {PropTypes} from 'react';
```

Comment:
The stable does have some different functionality from a clutch view - it displays the # of drakes instead of the # of pages, it displays static drakes instead of hatching drakes. I don't see a way to cleanly combine them, but please let me know if that's the approach you would take